internal/aws/awsutil/conn.go

```diff
@@ -181,6 +184,18 @@ func GetAWSConfigSession(logger *zap.Logger, cn ConnAttr, cfg *AWSSessionSetting
 		Endpoint:               aws.String(cfg.Endpoint),
 		HTTPClient:             http,
 	}
+	defaultCredProviders := defaults.CredProviders(config, defaults.Handlers())
```

Comment:
Can't have override singleton here since this is an internal package and can't have the intern repo update values for here.